geneticengine/core/representations/grammatical_evolution/dynamic_structured_ge.py

```diff
@@ -297,45 +296,52 @@ def random_float(self, min: float, max: float, prod: str = "") -> float:
 
 def create_tree(g: Grammar, ind: Genotype, depth: int) -> TreeNode:
     rand: Source = DynamicStructuredListWrapper(ind)
-    return random_node(rand, g, depth, g.starting_symbol, method=Grow)
```

Comment:
I think grow should also be available, and not only PI grow (you said Nuno uses this, which should mean we need to do it as well, but it must be clearly documented, and other methods should be available). Let's discuss this.